### DISCUSSION TITLE
Transparent text color for a Label with Background(or some parent layout)

### DISCUSSION THREAD

**Discussion by mhrastegari** [dotnet/maui#14511] [Q&A]
Is there a way to achive something like this in Xaml?

![image](https://i.stack.imgur.com/yTOAb.jpg)

**Comment by xtuzy** [comment]
google native android how to do it, see https://stackoverflow.com/questions/19974078/android-transparent-text, it is not out of the box, so you need custom native view.
If your text is simple, maybe you can use graphicsview to draw

**Reply by mhrastegari** [reply]
By GraphicsView you mean something like this with SkiaSharp?
https://learn.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/graphics/skiasharp/basics/text

**Reply by xtuzy** [reply]
we know android use skia, if android can draw, that means skiasharp can draw it, but skiasharp is big, so you can try graphicsview https://learn.microsoft.com/en-us/dotnet/maui/user-interface/graphics/?view=net-maui-7.0 , if it can't do, then you need skiasharp
